phpunit/data/themedir1/fse/experimental-theme.json

```diff
@@ -0,0 +1,33 @@
+{
+	"settings": {
+		"root": {
```

Comment:
@nosolosw – is it `defaults` or `root` now? According to [docs](https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/themes/theme-json.md), it's `defaults` but it didn't work in my testing.